docs/userguide/declarative_config.rst

```diff
@@ -193,28 +188,30 @@ platforms                       platform           list-comma
 provides                                           list-comma
 requires                                           list-comma
 obsoletes                                          list-comma
-==============================  =================  =================  =============== =====
+==============================  =================  =================  =============== =======================
 
-.. note::
-    A version loaded using the ``file:`` directive must comply with PEP 440.
-    It is easy to accidentally put something other than a valid version
-    string in such a file, so validation is stricter in this case.
```

Comment:
Moved this around because I find it was scattering into pieces informations about a single keyword and breaking the reading flow. Also, all those grey notes boxes aren't that pretty.

Reply:
> Also, all those grey notes boxes aren't that pretty.

Honestly I can't wait for the theme to be switched to furo.